src/sst/elements/osseous/examples/vecshiftreg/rtlevent.h

```diff
@@ -0,0 +1,73 @@
+// Copyright 2009-2020 NTESS. Under the terms
+// of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Copyright (c) 2009-2020, NTESS
+// All rights reserved.
+//
+// Portions are copyright of other developers:
+// See the file CONTRIBUTORS.TXT in the top level directory
+// the distribution for more information.
+//
+// This file is part of the SST software package. For license
+// information, see the LICENSE file in the top level directory of the
+// distribution.
+
+#ifndef _RTLEVENT_H
+#define _RTLEVENT_H
+
+#include "uint.h"
+#include "VecShiftRegister_O1.h"
+#include <string>
+#include <sst/core/event.h>
+
+namespace SST {
+    namespace RtlComponent { 
+
+class RTLEvent : public SST::Event {
+public:
+    typedef std::vector<char> dataVec;
+    bool update_inp, update_ctrl, update_registers, verbose, done_reset,
+         update_eval_args, sim_done;
+    uint64_t sim_cycles;
+    RTLEvent() : SST::Event() { }
+    dataVec payload;
+
+    void input_sigs(VecShiftRegister*);
+    void control_sigs(VecShiftRegister*);
+    void UpdateRtlSignals(void*, VecShiftRegister*, uint64_t&);
+    void *inp_ptr, *ctrl_ptr;
+
+    void serialize_order(SST::Core::Serialization::serializer &ser)  override {
```

Comment:
Serialization needs to serialize every member of the class, not just the payload.